website/content/api-docs/secret/identity/mfa/duo.mdx

```diff
@@ -11,23 +11,23 @@ This endpoint defines an MFA method of type Duo.
 
 | Method | Path                           |
 | :----- | :----------------------------- |
-| `POST` | `/identity/mfa/method/duo/:id` |
+| `POST` | `/identity/mfa/method/duo/:method_id` |
 
 ### Parameters
 
-- `id` `(string: "")` - Optional UUID to specify if updating an existing method.
+- `method_id` `(string: "")` - Optional UUID to specify if updating an existing method.
 
-- `username_format` `(string)` - A template string for mapping Identity names to MFA methods. Values to substitute should be placed in `{{}}`. For example, `"{{identity.entity.name}}"`. If blank, the Entity's Name field is used as-is.
+- `method_name` `(string: )` - The unique name identifier for this MFA method. Supported from Vault 1.13.0.
+
+- `username_format` `(string: )` - A template string for mapping Identity names to MFA methods. Values to substitute should be placed in `{{}}`. For example, `"{{identity.entity.name}}"`. If blank, the Entity's Name field is used as-is.
```

Comment:
❓ Are the defaults for `method_name` and `username_format` supposed to be an empty string, in other words `(string: "")`? There are a few additional examples of this elsewhere in the changes.

Reply:
Thanks for the feedback! Great catch. Fixed.